File: Lab1/coordinate_geometry.py
import math

x1 = int(input("enter x1: "))
y1 = int(input("enter y1: "))
x2 = int(input("enter x2: "))
y2 = int(input("enter y2: "))

distance = math.sqrt(math.pow(x1-x2, 2) + math.pow(y1-y2, 2))
print(f"Distance: {distance}")

File: Lab1/coordinate_geometry.cpp
#include <cmath>
#include <iostream>

int main() {
  int x1, x2, y1, y2;
  std::cout << "Enter x1, y1, x2, y2: ";
  std::cin >> x1 >> y1 >> x2 >> y2;

  double distance = std::sqrt(std::pow(x1 - x2, 2) + std::pow(y1 - y2, 2));
  std::cout << "Distance: " << distance << "\n";

  return 0;
}